media/base/audio_codecs.cc

```diff
@@ -62,6 +66,10 @@ std::string GetProfileName(AudioCodecProfile profile) {
       return "unknown";
     case AudioCodecProfile::kXHE_AAC:
       return "xhe-aac";
+    case AudioCodecProfile::kIAMF_SIMPLE:
```

Comment:
Let's create a bug to create tests for all IAMF codec and profile.

Reply:
b/335487480